src/Meshes/WarpCubetoSphere.jl

```diff
@@ -0,0 +1,62 @@
+function cubed_sphere_warp(
+    ::EquiangularSphereMesh,
+    a,
+    b,
+    c,
+    R = max(abs(a), abs(b), abs(c)),
+)
+
+    function f(sR, ξ, η)
+        X, Y = tan(π * ξ / 4), tan(π * η / 4)
+        ζ1 = sR / sqrt(X^2 + Y^2 + 1)
+        ζ2, ζ3 = X * ζ1, Y * ζ1
+        ζ1, ζ2, ζ3
+    end
+
+    fdim = argmax(abs.((a, b, c)))
+    if fdim == 1 && a < 0
+        # (-R, *, *) : formulas for Face I from Ronchi, Iacono, Paolucci (1996)
+        #              but for us face II of the developed net of the cube
+        x1, x2, x3 = f(-R, b / a, c / a)
+    elseif fdim == 2 && b < 0
+        # ( *,-R, *) : formulas for Face II from Ronchi, Iacono, Paolucci (1996)
+        #              but for us face III of the developed net of the cube
+        x2, x1, x3 = f(-R, a / b, c / b)
+    elseif fdim == 1 && a > 0
+        # ( R, *, *) : formulas for Face III from Ronchi, Iacono, Paolucci (1996)
+        #              but for us face IV of the developed net of the cube
+        x1, x2, x3 = f(R, b / a, c / a)
+    elseif fdim == 2 && b > 0
+        # ( *, R, *) : formulas for Face IV from Ronchi, Iacono, Paolucci (1996)
+        #              but for us face I of the developed net of the cube
+        x2, x1, x3 = f(R, a / b, c / b)
+    elseif fdim == 3 && c > 0
+        # ( *, *, R) : formulas for Face V from Ronchi, Iacono, Paolucci (1996)
+        #              and the same for us on the developed net of the cube
+        x3, x2, x1 = f(R, b / c, a / c)
+    elseif fdim == 3 && c < 0
+        # ( *, *,-R) : formulas for Face VI from Ronchi, Iacono, Paolucci (1996)
+        #              and the same for us on the developed net of the cube
+        x3, x2, x1 = f(-R, b / c, a / c)
+    else
+        error(
+            "invalid case for cubed_sphere_warp(::EquiangularCubedSphere): $a, $b, $c",
+        )
+    end
+
+    return x1, x2, x3
+end
+
+function cubed_sphere_warp(
+    ::EquidistantSphereMesh,
+    a,
+    b,
+    c,
+    R = max(abs(a), abs(b), abs(c)),
+)
+    r = hypot(a, b, c)
+    x1 = R * a / r
+    x2 = R * b / r
+    x3 = R * c / r
+    return x1, x2, x3
+end
```

Comment:
Can you at least acknowledge authorship of these functions or the repo from which you grabbed them? At least in a comment and/or documentation

Reply:
@simonbyrne , @valeriabarra : For code obtained from ClimateMachine.jl, should we use the url? e.g.: https://github.com/CliMA/ClimateMachine.jl/blob/master/src/Numerics/Mesh/Topologies.jl

Reply:
All code for both repos is under the same copyright holder and worked on by overlapping authors so we don't need to do much per file / function, but we should create a note in the documentation and README citing the ClimateMachine.jl repo and (eventual) paper: see https://github.com/CliMA/ClimaCore.jl/issues/191.